src/plugin.py

```diff
@@ -356,6 +356,29 @@ def _get_not_added_free_games(owned_games):
         finally:
             self.local_games_called = True
 
+
+    async def get_game_time(self, game_id, context):
+        log.info('game_id: ' + game_id)
+        if game_id == "5272175": # Overwatch
+            log.debug("Fetching playtime for Overwatch...")
+            player_data = await self.backend_client.get_ow_player_data()
+            if player_data is None:
+                log.error('No Overwatch profile found.')
+                return GameTime(game_id, None, None)
+            if player_data['private'] == True:
+                log.info('Unable to get data as Overwatch profile is private.')
+                return GameTime(game_id, None, None)
+            qp_time = player_data['playtime']['quickplay']
+            game_time_minutes = None
+            if qp_time.count(':') == 1: # minutes and seconds
+                match = re.search('(?:(?P<m>\\d+):)(?P<s>\\d+)', qp_time)
+                game_time_minutes = int(qp_time.group('m'))
+            elif qp_time.count(':') == 2: # hours, minutes and seconds
+                match = re.search('(?:(?P<h>\\d+):)(?P<m>\\d+)', qp_time)
+                game_time_minutes = int(match.group('h')) * 60 + int(match.group('m'))
+            log.info('Overwatch quickplay playtime [minutes]: ' + str(game_time_minutes) + " (" + qp_time + ")")
```

Comment:
qp_time is already added, right?

```suggestion
            log.info(f'Overwatch quickplay playtime [minutes]: {game_time_minutes}')
```

Reply:
Added tests. :) The gametime is for the entire account, not by played heroes. I covered public, private and non-existing profiles. But I have no idea how public profiles that haven't played a single game look like... I believe this to be a edge case I wouldn't cover right now. ;)